src/character.cpp

```diff
@@ -243,30 +243,31 @@ void Character::recalc_sight_limits()
     // (A player will never have more than one night vision trait.)
     sight_boost_cap = 12;
     // Debug-only NV, by vache's request
-    if (has_trait("DEBUG_NIGHTVISION")) {
+    if (has_active_mutation("DEBUG_NIGHTVISION")) {
```

Comment:
If you're using a debug tool you can just debug it away.  I'm not sure this is any faster than debug menu -> mutations-> Debug Vision -> toggle.